### PR TITLE
Features/payment service

### PR DESCRIPTION
**Pull request by kenwaitam**

### CODE REVIEW

File: src/PaymentAPI/EventHandler.cs [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+			_messageHandler.Stop();
+		}
+
+		public async Task<bool> HandleMessageAsync(MessageTypes messageType, string message)
+		{
+			JObject messageObject = MessageSerializer.Deserialize(message);
+			try
+			{
+				switch (messageType)
+				{
```
Comment (richardbos09) [inline src/PaymentAPI/EventHandler.cs]: You have a switch here that's doing nothing. For what reason is there an EventHandler, if it's not expecting anything?

File: src/PaymentAPI/Properties/launchSettings.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:5900/",
```
Comment (richardbos09) [inline src/PaymentAPI/Properties/launchSettings.json:6]: Is it post number 5900 or 6000?

File: src/PaymentAPI/Properties/launchSettings.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      }
+    },
+    "PaymentAPI": {
+      "commandName": "Project",
+      "launchBrowser": true,
+      "launchUrl": "api/values",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      },
+      "applicationUrl": "http://localhost:6000/"
```
Comment (richardbos09) [inline src/PaymentAPI/Properties/launchSettings.json]: See previous comment

File: src/PaymentAPI/appsettings.Production.json [hunk 4]
```diff
@@ -0,0 +1,10 @@
+﻿{
+  "RabbitMQ": {
+    "Host": "rabbitmq",
+    "Username": "rabbitmquser",
+    "Password": "DEBmbwkSrzy9D1T9cJfa"
+  },
+  "ConnectionStrings": {
```
Comment (richardbos09) [inline src/PaymentAPI/appsettings.Production.json]: Remove this if you not gonna use a database

**Review by richardbos09 — CHANGES_REQUESTED** [review]
See comments in Files changed

File: src/PaymentAPI/Properties/launchSettings.json (hunk 2 above)
Comment (kenwaitam) [reply src/PaymentAPI/Properties/launchSettings.json:6]: ok

**Review by richardbos09 — APPROVED** [review]

**Pull request merged**